test/e2e/nns_update_timestamp_test.go

```diff
@@ -0,0 +1,37 @@
+package e2e
+
+import (
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	framework "github.com/operator-framework/operator-sdk/pkg/test"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+)
+
+var _ = Describe("NNS LastSuccessfulUpdateTime", func() {
+	Context("when updating nns", func() {
+		It("timestamp should be changed", func() {
+			for _, node := range nodes {
+				key := types.NamespacedName{Name: node}
+				originalTime := nodeNetworkState(key).Status.LastSuccessfulUpdateTime
+
+				configMap, err := framework.Global.KubeClient.CoreV1().ConfigMaps("nmstate").Get("nmstate-config", metav1.GetOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				interval, err := strconv.Atoi(configMap.Data["node_network_state_refresh_interval"])
+				Expect(err).ToNot(HaveOccurred())
+
+				// Give enough time for the NNS to be updated (3 interval times)
+				timeout := time.Duration(interval*3) * time.Second
```

Comment:
The math does not make sense here. The duration should have the unit (Second) in it. In this state, the duration (let's say its 3 milisecond, but it can be 3 anything) is being multiplied by some internal value hidden in Second.

It should be `time.Duration(3*interval*time.Second)`